File: CTDK_Python/082/_082.py
from math import sin


x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
i = 1
while(i <= n):
    t = t * sin(x)
    s = s + t
    i = i + 1

print("Tong la: ", s)

File: CTDK/082/082.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	float t = 1;
	int i = 1;
	while (i <= n)
	{
		t = t * sin(x);
		s = s + t;
		i++;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}